packages/app-page-builder/src/render/components/WebsiteScripts.tsx

```diff
@@ -1,6 +1,19 @@
 import React, { useEffect, useMemo } from "react";
 import { Helmet } from "react-helmet";
 
+const wrapWithRenderCondition = (innerHTML: string) => {
```

Comment:
Curious about one thing here...

Couldn't we _not_ do this, and, instead, at the beginning of the `WebsiteScripts` component, just do the `if (!("__PS_RENDER__" in window))` check and bail if so?

So:

```tsx
const WebsiteScripts: React.FC<WebsiteScriptsProps> = ({ headerTags, footerTags }) => {
    // `WebsiteScripts` component doesn't do anything if we're
    // in the middle of a prerendering process.
    if ("__PS_RENDER__" in window) {
        return null;
    }
    
    const htmlHeadTags = useMemo(() => {
        return parseHeaderTags(headerTags);
    }, [headerTags]);
    useEffect(() => {
        const htmlFooterTags = parseFooterTags(footerTags);
        appendToBody(htmlFooterTags);
        return () => {
            removeFromBody(htmlFooterTags);
        };
    }, [footerTags]);
    return <Helmet>{htmlHeadTags?.map(element => element)}</Helmet>;
};
```

Reply:
If I understand rendering process right - no
As I see it, on prerender we are getting a final HTML which is saved and served latter as a static html file (without react).
So we need to add these scripts on prerender, but we cannot allow them to be executed (as it causes a DOM changes which are not intended).

On other hand we need them to be executable on preview page (so these can be tested before publishing)...
We may consider smth like a POST prerender html manipulation to add these scripts... but it will change a current cms flow

Reply:
@neatbyte-vnobis the HTML is hydrated after being served to the browser, and React starts again.

Reply:
Initially, I wasn't sure if there was some additional website scripts-related logic involved that I wasn't aware of. 

But, from what I can see now, there isn't. So, the above suggestion should work fine and the code can be simplified (`wrapWithRenderCondition` can be removed).